Volume/Views/MainView/HomeList.swift

```diff
@@ -257,6 +264,7 @@ struct HomeList: View {
     var otherArticlesSection: some View {
         Group {
             Header("Other Articles").padding()
```

Comment:
put `.padding()` in a new line, similar to the Headers for "The Big Read" and "Following"